From 9a1f298f8bc92f21b09bd7ef396c2fc6544ce9aa Mon Sep 17 00:00:00 2001
From: Johan Hovold <jhovold@gmail.com>
Date: Sun, 16 May 2010 15:06:55 +0200
Subject: USB: visor: increase bulk-out buffer size
Patch-mainline: v2.6.35
Git-commit: 9a1f298f8bc92f21b09bd7ef396c2fc6544ce9aa

Increase the bulk-out buffer size to avoid any regression in throughput
after replacing the old writing scheme which used dynamic buffers (e.g.
up to 2k).

256b has been determined to be a good choice for several drivers
including ftdi_sio which used to have a more or less identical write
implementation.

Signed-off-by: Johan Hovold <jhovold@gmail.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/usb/serial/visor.c |    3 +++
 1 file changed, 3 insertions(+)

--- a/drivers/usb/serial/visor.c
+++ b/drivers/usb/serial/visor.c
@@ -187,6 +187,7 @@ static struct usb_serial_driver handspri
 	.usb_driver =		&visor_driver,
 	.id_table =		id_table,
 	.num_ports =		2,
+	.bulk_out_size =	256,
 	.open =			visor_open,
 	.close =		visor_close,
 	.throttle =		usb_serial_generic_throttle,
@@ -207,6 +208,7 @@ static struct usb_serial_driver clie_5_d
 	.usb_driver =		&visor_driver,
 	.id_table =		clie_id_5_table,
 	.num_ports =		2,
+	.bulk_out_size =	256,
 	.open =			visor_open,
 	.close =		visor_close,
 	.throttle =		usb_serial_generic_throttle,
@@ -227,6 +229,7 @@ static struct usb_serial_driver clie_3_5
 	.usb_driver =		&visor_driver,
 	.id_table =		clie_id_3_5_table,
 	.num_ports =		1,
+	.bulk_out_size =	256,
 	.open =			visor_open,
 	.close =		visor_close,
 	.throttle =		usb_serial_generic_throttle,
